Navigation

butalas opened the pull request:
### Description
The branch connects the screens together and makes links with React Navigation. 

### Context
Connects the screens together with React Navigation. Sanjana made the `DriverSearch` updates so I didn't make the changes to the page on my branch. I linked the pages that could only be linked and were present, since some weren't there and others didn't have the `PrimaryButton` component implemented. 

### Issue
#58 

### Screenshot
N/A* (only connecting screens)

ijemmao commented on App.js:
```diff
@@ -44,39 +96,39 @@ tabBarOptions: {
 export default createAppContainer(TabNavigator);
 
-class App extends React.Component {
+// class App extends React.Component {
```
Removed commented out lines

ijemmao commented on screens/Register.js:
```diff
@@ -1,33 +1,44 @@
       <View style={styles.container}>
-        <View>
-          <View>
+        <View style={styles.peachBox}>
```
Use the `OrangeComponent` that @architaharathi created

butalas commented:
I just removed those commented out lines with the `App` class and configured the `Register.js` page to work with the `OrangeBackground` component Archita made. I also fixed up some of the margins so they are all related with `Dimensions`.
